### PR TITLE
Java: Add command GeoHash

### PR DESCRIPTION
**Pull request by tjzhang-BQ**
Adds GeoHash and tests

### CODE REVIEW

File: java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java [hunk 1]
```diff
@@ -78,4 +78,22 @@ CompletableFuture<Long> geoadd(
      * }</pre>
      */
     CompletableFuture<Double[][]> geopos(String key, String[] members);
+
+    /**
+     * Returns the GeoHash strings representing the positions of all the specified members in the
```
Comment (Yury-Fridlyand) [inline java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java:83]: Wrap members with `code`.

Comment (Yury-Fridlyand) [reply java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java:83]: And maybe GeoHash too

File: java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java [hunk 2]
```diff
@@ -78,4 +78,22 @@ CompletableFuture<Long> geoadd(
...
+     * sorted set stored at <code>key</code>.
+     *
+     * @see <a href="https://valkey.io/commands/geohash">valkey.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @param members The list of members whose GeoHash strings are to be retrieved.
+     * @return A list of GeoHash strings representing the positions of the specified members stored at
```
Comment (Yury-Fridlyand) [inline java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java:89]: list -> array
Don't duplicate description. Feel free to reword both.

File: java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java [hunk 3]
```diff
@@ -78,4 +78,22 @@ CompletableFuture<Long> geoadd(
...
+     *     <code>key</code>. If a member does not exist in the sorted set, a <code>null</code> value
+     *     is returned for that member.
+     * @example
+     *     <pre>{@code
+     * String[] result = client.geohash("mySortedSet", new String[] {"Palermo", "Catania", "NonExisting"}).get();
+     * System.out.println(Arrays.toString(result));
```
Comment (Yury-Fridlyand) [inline java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java:95]: What does it print?

File: java/integTest/src/test/java/glide/TransactionTestUtilities.java [hunk 4]
```diff
@@ -455,14 +455,16 @@ private static Object[] geospatialCommands(BaseTransaction<?> transaction) {
...
+                .geopos(geoKey1, new String[] {"Palermo", "Catania"})
+                .geohash(geoKey1, new String[] {"Palermo", "Catania", "NonExisting"});
 
         return new Object[] {
             2L, // geoadd(geoKey1, Map.of("Palermo", ..., "Catania", ...))
             new Double[][] {
                 {13.36138933897018433, 38.11555639549629859},
                 {15.08726745843887329, 37.50266842333162032},
             }, // geopos(new String[]{"Palermo", "Catania"})
+            new String[] {"sqc8b49rny0", "sqdtr74hyu0", null}
```
Comment (Yury-Fridlyand) [inline java/integTest/src/test/java/glide/TransactionTestUtilities.java:467]: Add a comment

Comment (tjzhang-BQ) [reply java/integTest/src/test/java/glide/TransactionTestUtilities.java]: Applied the changes to the docs/comments in https://github.com/Bit-Quill/glide-for-redis/pull/275/commits/52d70471ed7790557d6ca681ecb9b6247dea9b50

**Review by Yury-Fridlyand — APPROVED** [review]

**Pull request merged**